### PR TITLE
System test: quadlet kube basic test

### PR DESCRIPTION
**Pull request by ygalblum**
#### Does this PR introduce a user-facing change?
No

```release-note
None
```

### CODE REVIEW

File: test/system/252-quadlet.bats [hunk 1]
```diff
@@ -283,4 +283,43 @@ EOF
...
+  labels:
+    app: test
+  name: test_pod
+spec:
+  containers:
+  - command:
+    - top
+    image: $IMAGE
+    name: test
+    resources: {}
```
Comment (rhatdan) [inline test/system/252-quadlet.bats:302]: nit: this is not needed.

**Comment by rhatdan** [comment]
LGTM
@alexlarsson @vrothberg @edsantiago PTAL

File: test/system/252-quadlet.bats (hunk 1 above)
Comment (ygalblum) [reply test/system/252-quadlet.bats]: Fixed

**Review by vrothberg — APPROVED** [review]
/lgtm
/hold

restarted the flaked jobs

**Pull request merged**